scripts/test/Muon/help_widget_presenter_test.py

```diff
@@ -25,11 +25,19 @@ def setUp(self):
     def tearDown(self):
         self.view = None
 
-    @mock.patch('Muon.GUI.Common.help_widget.help_widget_view.ManageUserDirectories')
-    def test_that_manage_directories_button_clicked_opens_directory_manager(self, ManageUserDirectories_mock):
+    @mock.patch('Muon.GUI.Common.help_widget.help_widget_view.manageuserdirectories.OpenManageUserDirectories')
+    def test_that_manage_directories_button_clicked_opens_directory_manager(self, OpenManageUserDirectories_mock):
         self.view.manage_user_dir_button.clicked.emit(True)
 
-        ManageUserDirectories_mock.openUserDirsDialog.assert_called_once_with(self.view)
+        OpenManageUserDirectories_mock.open_mud.assert_called_once_with(self.view)
+
+    @mock.patch('Muon.GUI.Common.help_widget.help_widget_view.manageuserdirectories.OpenManageUserDirectories')
+    def test_that_manage_directories_button_does_not_open_multiple_times(self, OpenManageUserDirectories_mock):
+        OpenManageUserDirectories_mock.currently_open_mud = True
+        self.view.manage_user_dir_button.clicked.emit(True)
+
+        # Checking that the value of currently_open_mud hasn't changed.
+        self.assertTrue(OpenManageUserDirectories_mock.currently_open_mud)
```

Comment:
So if a MUD is already open this wouldn't detect us trying to open another (as it would still be true)
Could we do something like

`OpenManagerUserDirectories_mock.currently_open_mud.raise_().assert_called_once()`